.github/workflows/super-linter.yml

```
@@ -0,0 +1,40 @@
+          VALIDATE_PYTHON_BLACK: true
+          VALIDATE_PYTHON_FLAKE8: true
+          VALIDATE_ALL_CODEBASE: true
+          VALIDATE_YAML: true
+          VALIDATE_MAKEFILE: true
+          VALIDATE_TERRAFORM_FMT: true
+          VALIDATE_BASH: true
+          FIX_MARKDOWN_PRETTIER: true
+          FIX_YAML_PRETTIER: true
+          FIX_PYTHON_BLACK: true
+          FIX_ENV: true
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)